### PR TITLE
FedCM Chrome 120 updates

### PR DESCRIPTION
**Pull request by agektmr**
FedCM Chrome 120 updates

### CODE REVIEW

**Comment by rachelandrew** [comment]
I just added myself to this to keep track, but to be included in the 120 blog post this will need to be live by 9am PT, as usually the beta goes out shortly after that.

**Comment by agektmr** [comment]
Alexandra gave it an initial pass at it already. Hopefully there is not much to edit further, but I'll leave it to her to decide whether to publish this before or after the beta announcement.

File: site/en/blog/fedcm-chrome-120-updates/index.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+```
+
+Alternatively, call the JavaScript API `IdentityProvider.logout()` from the IdP
+origin:
+
+```js
+navigator.login.setStatus("logged-out")
+```
+
+These calls record the user's login status as "sign-out". When the user's login
```
Comment (heyawhite) [inline site/en/blog/fedcm-chrome-120-updates/index.md:120]:
```suggestion
These calls record the user's login status as "signed-out." When the user's login
```

File: site/en/blog/fedcm-chrome-120-updates/index.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+### What if the user session expires? Let the user sign in through a dynamic login flow!
+
+Even though the IdP keeps informing the user's login status to the browser, it
+could be out of sync, such as when the session expires. The browser tries to
+send a credentialed request to the accounts list endpoint when the login status
+is "logged-in", but the server returns no accounts because the session is no
+longer available. In such a scenario, the browser can dynamically let the user
+sign in to the IdP through a popup window.
+
+The FedCM dialog will display a message as shown in the following image:
```
Comment (heyawhite) [inline site/en/blog/fedcm-chrome-120-updates/index.md:145]:
```suggestion
The FedCM dialog displays a message suggesting a sign in, as shown in the following image.
```

**Review by heyawhite — APPROVED** [review]
Nits which I'll merge

**Pull request merged**